pages/docs/quick-sqlite/better-sqlite3.mdx

```diff
@@ -45,13 +45,30 @@ const db = drizzle(sqlite);
 const result = await db.select().from(users);
 ```
 
+Unless you plan on writing every SQL query by hand, a table declaration is helpful:
+```typescript copy
+import { sql } from "drizzle-orm";
+import { text, integer, sqliteTable } from "drizzle-orm/sqlite-core";
+
+const users = sqliteTable('users', {
+  id: text('id'),
+  textModifiers: text('text_modifiers').notNull().default(sql`CURRENT_TIMESTAMP`),
+  intModifiers: integer('int_modifiers', { mode: 'boolean' }).notNull().default(false),
+});
+```
+For more details about column types, see the **[SQLite column types in Drizzle.](/docs/column-types/sqlite)**
+
+<Callout type="info" emoji="ℹ️">
+  If your `db.insert()` returns data, append `all()` to your query, otherwise append `run()`
+</Callout>
+
 If you want to use **sync** APIs:
 ```typescript copy
-import { drizzle } from 'drizzle-orm/bun-sqlite';
-import { Database } from 'bun:sqlite';
+import { drizzle, BetterSQLite3Database } from 'drizzle-orm/bun-sqlite';
```

Comment:
we've removed type annotations of the database across the docs, I'll merge the pr and remove them myself